.circleci/config.yml

```diff
@@ -36,21 +36,21 @@ jobs:
       - attach_workspace:
           at: ~/react-native-testing-library
       - run: |
+          cd packages/react-native-testing-library
```

Comment:
linting and type checks should go from the root, please remove and adjust configuration if needed